packages/nimble/R/genCpp_sizeProcessing.R

```diff
@@ -368,7 +368,7 @@ sizeRecyclingRuleRfunction <- function(code, symTab, typeEnv) {
     code$sizeExprs <- newSizeExprs
     code$type <- 'double' ## will need to look up from a list
     code$nDim <- 1
-    code$toEigenize <- TRUE
```

Comment:
No, that was actually an incidental catch.  It wasn't the source of the bug in question.  BTW I think the global view of the toEigenize logic is a little more complicated that we discussed but still separable.